src/lib/zf/private/reactor.c

```diff
@@ -152,15 +152,15 @@ zf_reactor_handle_rx_ref_discard(struct zf_stack* st, int nic, ef_vi *vi,
 
 
 ZF_HOT static void
```

Comment:
Perhaps document that efpkt is a packet pointer retrieved through `efct_vi_rxpkt_get()`?